site-client/aktin-fhir-mii-report-client.py

```diff
@@ -193,6 +196,10 @@ def execute_pat_year_queries():
     while cur_year <= last_year:
         pat_ids = set()
         query = f'/Encounter?date={str(cur_year)}&_count=500'
+
+        if pat_year_with_facility:
+            query = f'/Encounter?type=http://fhir.de/CodeSystem/Kontaktebene|einrichtungskontakt&date={str(cur_year)}&_count=500'
```

Comment:
The query looks fine for me and is working as expected on our server. But I am not sure if | needs to be escaped here.